### ISSUE TITLE
[Bug] ARTrackableManager.trackablesChanged is null after creating the manager from script

### ISSUE THREAD

**Issue by KyryloKuzyk** [Unity-Technologies/arfoundation-samples#1121]
**Describe the bug**
ARTrackableManager.trackablesChanged is null after creating the manager from a script. And there is no way to assign a non-null value to trackablesChanged because its setter is private.

**To Reproduce**
```csharp
#if TEST_FRAMEWORK_INSTALLED
using NUnit.Framework;
using UnityEngine;
using UnityEngine.XR.ARFoundation;

public class EditModeTests {
    #if AR_FOUNDATION_6_0_OR_NEWER
    [Test]
    public void TrackablesChangedNullRefBug() {
        var go = new GameObject();
        try {
            var planeManager = go.AddComponent<ARPlaneManager>();
            Assert.IsNotNull(planeManager.trackablesChanged); // this assertion fails
        } finally {
            Object.DestroyImmediate(go);
        }
    }
    #endif
}
#endif
```

**Proposed fix**
Set the default value in field declaration.
```csharp
public UnityEvent<ARTrackablesChangedEventArgs<TTrackable>> trackablesChanged { get; private set; } = new UnityEvent<ARTrackablesChangedEventArgs<TTrackable>>();
```

**Setup**
 - ARFoundation 6.0.0-pre.4

**Comment by andyb-unity** [comment]
Thanks @KyryloKuzyk. We're on it.

**Comment by andyb-unity** [comment]
We just merged this in. Will be in 6.0.0-pre.5. Our current thinking is to cut at least one more pre-release before we ship 6.0.0.

**Comment by andyb-unity** [comment]
Fixed in 6.0.0-pre.5.
